catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -223,6 +240,7 @@ public UpdateStorageResponse version(
             .map(ContentItem::getMetacard)
             .filter(Objects::nonNull)
             .filter(isNotVersionNorDeleted)
+            .filter(this::isNotBlackListedUpdate)
```

Comment:
We could split the config to allow for versioning on the content separately but not sure there's a use case for skipping versioning on the direct metacard updates, but not the content updates.